reana_server/rest/users.py

```diff
@@ -283,15 +290,46 @@ def get_me():
               {
                 "error": "User not logged in"
               }
+        403:
+          description: >-
+            Request failed. User token not valid.
+          examples:
+            application/json:
+              {
+                "message": "Token is not valid."
+              }
+        500:
+          description: >-
+            Request failed. Internal server error.
+          examples:
+            application/json:
+              {
+                "message": "Internal server error."
+              }
     """
-    if current_user.is_authenticated:
-        me = _get_user_from_invenio_user(current_user.email)
-        return (
-          jsonify({'email': me.email, 'reana_token': me.access_token,
-                   'full_name': me.full_name, 'username': me.username}), 200
-        )
-    else:
-        return jsonify({'error': 'User not logged in'}), 401
+    try:
+        me = None
+        if current_user.is_authenticated:
+            me = _get_user_from_invenio_user(current_user.email)
+        elif "access_token" in request.args:
+            me = get_user_from_token(request.args.get('access_token'))
+
+        if me:
+            return (jsonify({'email': me.email,
+                             'reana_token': me.access_token,
+                             'full_name': me.full_name,
+                             'username': me.username}), 200)
+        else:
+            return jsonify({'error': 'User not logged in'}), 401
```

Comment:
Client expects all errors to have a key called `message`.
```console
$ reana-client -l DEBUG ping
...
[2020-02-25 10:58:07,177] p85169 {/Users/rodrigdi/.virtualenvs/reana-integration/lib/python3.6/site-packages/urllib3/connectionpool.py:959} DEBUG - Starting new HTTPS connection (1): 192.168.99.134:30443
[2020-02-25 10:58:07,193] p85169 {/Users/rodrigdi/.virtualenvs/reana-integration/lib/python3.6/site-packages/urllib3/connectionpool.py:437} DEBUG - https://192.168.99.134:30443 "GET /api/me?access_token=COWwxGjGDtCY_kvHLmkul43V0asldkfjasdlkfa HTTP/1.1" 401 36
[2020-02-25 10:58:07,196] p85169 {/Users/rodrigdi/integration/reanahub/reana-client/reana_client/cli/ping.py:48} DEBUG - Traceback (most recent call last):
  File "/Users/rodrigdi/integration/reanahub/reana-client/reana_client/api/client.py", line 40, in ping
    .get_me(access_token=access_token).result())
  File "/Users/rodrigdi/.virtualenvs/reana-integration/lib/python3.6/site-packages/bravado/http_future.py", line 271, in result
    swagger_result = self._get_swagger_result(incoming_response)
  File "/Users/rodrigdi/.virtualenvs/reana-integration/lib/python3.6/site-packages/bravado/http_future.py", line 124, in wrapper
    return func(self, *args, **kwargs)
  File "/Users/rodrigdi/.virtualenvs/reana-integration/lib/python3.6/site-packages/bravado/http_future.py", line 302, in _get_swagger_result
    self.request_config.response_callbacks,
  File "/Users/rodrigdi/.virtualenvs/reana-integration/lib/python3.6/site-packages/bravado/http_future.py", line 352, in unmarshal_response
    raise_on_expected(incoming_response)
  File "/Users/rodrigdi/.virtualenvs/reana-integration/lib/python3.6/site-packages/bravado/http_future.py", line 418, in raise_on_expected
    swagger_result=http_response.swagger_result)
bravado.exception.HTTPUnauthorized: 401 Unauthorized: {'error': 'User not logged in'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/rodrigdi/integration/reanahub/reana-client/reana_client/cli/ping.py", line 40, in ping
    response = rs_ping(access_token)
  File "/Users/rodrigdi/integration/reanahub/reana-client/reana_client/api/client.py", line 55, in ping
    e.response.json()['message']))
KeyError: 'message'

[2020-02-25 10:58:07,196] p85169 {/Users/rodrigdi/integration/reanahub/reana-client/reana_client/cli/ping.py:49} DEBUG - 'message'
Could not connect to the selected REANA cluster server at https://192.168.99.134:30443:
'message'
```
Happening [here](https://github.com/reanahub/reana-client/pull/384/files#diff-4f27f08c3f46f0b755c1d5ac1a1e2dcaR55).

Reply:
Oh, good catch, should be fixed in r-client.